README.md

```diff
@@ -11,17 +11,30 @@ This app is live-coded every Sunday at 9am EST over at [https://twitch.tv/ChaelC
+| [Webpacker](https://github.com/rails/webpacker) | Webpack Integration in Rails  | Used to Build Vue                                                                                                                                                                         |
+| [Yarn](https://yarnpkg.com/)                    | JS Package Management         | -                                                                                                                                                                                         |
+| [Vue](https://vuejs.org/)                       | Front-end framework           | Add some reactivity                                                                                                                                                                       |
+| [Buefy](https://buefy.org/)                     | UI Component Library          | Based on [Bulma](https://bulma.io/)                                                                                                                                                       |
+| RDoc                                            | Documentation tool            | [View at rubydoc.info](https://rubydoc.info/github/ChaelCodes/HuntersKeepers/Hunter)                                                                                                      |
+| Inch                                            | Tracks Documentation Coverage | [![Inline docs](http://inch-ci.org/github/ChaelCodes/HuntersKeepers.svg?branch=master)](http://inch-ci.org/github/ChaelCodes/HuntersKeepers)                                              |
+| Code Climate                                    | Code Quality Solution         | [![Maintainability](https://api.codeclimate.com/v1/badges/67450702dbf049c335b0/maintainability)](https://codeclimate.com/github/ChaelCodes/HuntersKeepers/maintainability)                |
+| CodeShip                                        | Continuous Testing            | [![Codeship Status for ChaelCodes/HuntersKeepers](https://app.codeship.com/projects/c9216880-c9a8-0137-f66b-76a25f9e8127/status?branch=master)](https://app.codeship.com/projects/367780) |
+| Heroku                                          | Hosting                       | [Visit the App](https://hunters-keepers.herokuapp.com/hunters)                                                                                                                            |
+| Docker                                          | Development Environment       | -                                                                                                                                                                                         |
+
+## How to setup docker
```

Comment:
We have instructions on setting up Docker in the Contributing Guide. Directions for contributors are on line 8.

Reply:
sorry, I didn't see it, if I remove it will you accept the PR just for the table indentation if not you can close this PR?
Do you have something special where you want help? I didn't take the issues with the `good first issue` because I didn't understand the game rules

Reply:
The table indentation doesn't impact the table display in the README and makes it harder to work with the table. If someone added a table entry with a longer line, they'd have to adjust the whitespace in all previous entries.

I see what you mean about the issues. I'll spend some time today cleaning up issues and organizing them into projects.

Also, sorry for the delayed response!

Reply:
No worries, thank you!